src/symfc_alm/ridge.py

```diff
@@ -170,14 +174,26 @@ def run_auto(
         self._errors = np.zeros(len(self._alphas))
         for i, alpha in enumerate(self._alphas):
             self._fit(alpha, debias)
-            self._errors[i] = self._calc_error(alpha)
+            self._errors[i] = self._calc_error()
 
         self._opt_alpha = self._alphas[np.argmin(self._errors)]
         self._fit(self._opt_alpha, debias)
         self._psi = trans_prestandardize(self._coeff, self._scale, self._standardize)
 
     def _fit(self, alpha: float, debias: bool):
-        """Fit force constants.
+        r"""Fit force constants.
+
+        The regression coefficients for Ridge regression are calculated
+        using singular value decomposition as follows:
+
+        V(R^{T}R + \alpha I)^{-1}R^{T}b
+
+        where R is R = U @ Σ, and U, Σ, V are obtained from the
```

Comment:
Please avoid using Greek letter character.

Reply:
Thank you very much. I fixed it.